user/languages/objective-c.md

```diff
@@ -81,9 +87,32 @@ CocoaPods) are added explicitly to the Scheme. To do so:
 
 You will now have a new file in the **xcshareddata/xcschemes** directory
 underneath your Xcode project. This is the shared Scheme that you just
-configured. Check this file into your repository and xctool will be able to
+configured. Check this file into your repository and xcodebuild will be able to
 find and execute your tests.
 
+### Device Destinations
+
+To be able to run tests, xcodebuild needs to know which device to run them on, whether
+that's a real device (the Mac running xcodebuild or an iOS device connected to the Mac)
+or a simulator. When you run tests in Travis CI, you need to tell xcodebuild which
+simulator you want to use by specifying a **device destination**.
+
+Device destinations are strings that identify a particular device to use. You can pass
+them to xcodebuild by using the `-destination` flag. If you're using the default script
+in your Travis CI build, you can use the `xcode_destination` key in your .travis.yml:
+
+```
+xcode_destination: platform=iOS Simulator,OS=11.3,name=iPhone X
+```
+{: data-file=".travis.yml"}
+
+You can find more details about device destinations in the xcodebuild man page.
```

Comment:
I searched for this but only found unofficial postings of the contents of the xcodebuild man page. We could maybe link to one of those, but my preference had been to try to find something official, which I haven't been able to find.

Reply:
Fair enough.

Reply:
It's very disappointing to me that there isn't official online documentation about xcodebuild. There should be something we could link to here, but nothing from Apple seems to exist :(

Reply:
x-man-page://xcodebuild

:)

Reply:
I forgot about that. It'll only work if you click the link from a Mac (I think?) but maybe that's good enough?

Reply:
IMO, it's a pretty good link for this purpose because it should be current on the local machine and won't rot due to a 3rd party or even Apple moving a thing. Maybe we can mention in the link description that it's a link to a man page on MacOS?

Reply:
Maybe something like this?

"You can find more details about device destinations in the xcodebuild man page. If you're on your Mac, you can [click here] to view the man page in the Terminal app."

(GitHub doesn't allow links to the x-man-page scheme)